src/libs/DateUtils.ts

```diff
@@ -889,6 +889,22 @@ function getFormattedDateRangeForPerDiem(date1: Date, date2: Date): string {
     return `${format(date1, 'MMM d, yyyy')} - ${format(date2, 'MMM d, yyyy')}`;
 }
 
+/**
+ * Returns a formatted date range with the number of days in the range.
+ * Format: "YYYY-MM-DD to YYYY-MM-DD (X days)"
+ */
+function getFormattedSplitDateRange(translateParam: LocaleContextProps['translate'], startDate: string | undefined, endDate: string | undefined): string {
```

Comment:
Does not look like this function will be used anywhere else so can we keep it local to the component?

Reply:
Also, it is not a date util.